grammars/javascript.cson

```diff
@@ -354,8 +354,12 @@
     'name': 'entity.name.type.object.js.console'
   }
   {
-    'match': '\\.(warn|info|log|error|time|timeEnd|assert)\\b'
-    'name': 'support.function.js.console'
+    'match': '\\b(\\.)(assert|clear|debug|error|info|log|profile|profileEnd|time|timeEnd|warn)\\b(?=\\()'
```

Comment:
If you want to be really accurate you could use:

    (?<=\\bconsole)\\s*(\\.)\\s*(assert|clear|debug|error|info|log|profile|profileEnd|time|timeEnd|warn)\\b

to catch:

    console.log

and:

    console
            . log ()

but not:

    console.logs

or:

    notconsole . log